src/Polytopes/Polyhedron/properties.jl

```diff
@@ -933,4 +734,16 @@ julia> print_constraints(cube(3))
 6: x₃ ≦ 1
 ```
 """
-print_constraints(P::Polyhedron; trivial::Bool = false) = print_constraints(facets_as_halfspace_matrix_pair(P)...; trivial = trivial)
+print_constraints(P::Polyhedron; trivial::Bool = false, io::IO = stdout) = print_constraints(halfspace_matrix_pair(facets(P))...; trivial = trivial, io = io)
+
+print_constraints(H::Halfspace; trivial::Bool = false, io::IO = stdout) = print_constraints(H.a, [H.b]; trivial = trivial, io = io)
+
+function Base.show(io::IO, H::Halfspace)
+    n = length(H.a)
+    if iszero(H.a) && H.b >= 0
+        print(io, "The trivial Halfspace, R^$n")
+    else
+        print(io, "The Halfspace of R^$n described by\n")
+        print_constraints(H; io)
```

Comment:
I think this must be `io = io` on 1.4 if this is really a keyword argument.

Reply:
indeed
```suggestion
        print_constraints(H; io=io)
```